app/views/assessments/choose-evidence.html.erb

```diff
@@ -0,0 +1,211 @@
+<div class="govuk-grid-row">
```

Comment:
This does not look particularly easy to maintain. From our discussion, though, I understand you're planning to fix in future stories.